app/views/catalog/_form_basic_info.html.haml

```diff
@@ -52,7 +52,20 @@
                         :class                => "selectpicker")
           :javascript
             miqSelectPickerEvent('dialog_id', '#{url}')
-
+    - if @edit[:new][:st_prov_type] == "generic"
```

Comment:
@ZitaNemeckova this drop down should only be displayed when adding a new Catalog Item, user should not be allowed to change sub type on edit screen. On edit screen show subtype as read only text.